app/views.py

```diff
@@ -73,7 +74,7 @@ def post(self):
                                                 )
             db.session.add(new_bucket_list)
             db.session.commit()
-            return {"message": "You have created a new bucket list"}, 201
+            return {"created": bucket_list_data}, 201
```

Comment:
ideally the `created` field should not be there..you want to just `return bucket_list_data`